src/index.js

```diff
@@ -14,6 +14,23 @@ const getDefaultPayload = () => ({
   },
 });
 
+const getSessionHistory = () => {
+  const history = localStorage.getItem('history');
```

Comment:
Are we missing files in this commit?

Reply:
Because of usage of localStorage? It's a global object, residing in browser's `global` scope. Forgot to mock it for tests, should be good now.